libs/vkd3d-common/debug.c

```diff
@@ -281,11 +283,13 @@ const char *debugstr_w(const WCHAR *wstr)
 
 unsigned int vkd3d_env_var_as_uint(const char *name, unsigned int default_value)
 {
-    const char *value = getenv(name);
+    char value[VKD3D_PATH_MAX] = {};
```

Comment:
vkd3d_get_env_var should instead set `[0] = '\0';` as a defensive measure if env-var is not found instead of clearing out a few KiB every time.

Reply:
Done